support/mender-inventory-provides

```diff
@@ -10,6 +10,25 @@ set -e
 # Also submit device_type, which is in a separate file, not in the database. We don't know if the
 # file contains a newline, so it's important that this is done last, to avoid corrupting other
 # entries.
-grep '^device_type=' /var/lib/mender/device_type
+
+# Set default value for DeviceTypeFile
+default_device_type_file="/var/lib/mender/device_type"
+
+# Read mender.conf file to find the DeviceTypeFile key if it exists prioritizing /etc over /var/lib
+if grep -q '"DeviceTypeFile"' /etc/mender/mender.conf; then
+    device_type_file=$(grep '"DeviceTypeFile"' /etc/mender/mender.conf | awk -F ': ' '{print $2}' | tr -d '"' | tr -d ',')
+elif grep -q '"DeviceTypeFile"' /var/lib/mender/mender.conf; then
+    device_type_file=$(grep '"DeviceTypeFile"' /var/lib/mender/mender.conf | awk -F ': ' '{print $2}' | tr -d '"' | tr -d ',')
```

Comment:
Also, the order should be the opposite: `/var/lib/mender/mender.conf` contents should get precedence over `/etc/mender/mender.conf` contents

Reply:
Hello @lluiscampos, can you confirm we need to fix this paragraph in docs then?

> Any value set in the main configuration file /etc/mender/mender.conf is chosen, whether or not the setting appears in the fallback file /var/lib/mender/mender.conf

From https://docs.mender.io/client-installation/configuration#configuration

I used to define priority as I understoo /etc/ > /var/lib.

Reply:
You are totally right! (I double checked the code now) Thank you :clap: